docs/decisions/0008-zero-state-browsing-with-universal-link.rst

```diff
@@ -0,0 +1,164 @@
+in future tracking events, i.e. search events in the learner portal. 
+
+We also want to track how many users land on the logistration page but abandons the page. This could possibly be done through google analytics.
+
+Consequences
+------------
+
+* After a user requests course access and is granted a subsidy(ies), the learner portal will only display offerings associated with
+  the subsidy(ies). 
+* The JWT token will need to be refreshed to include the ``enterprise_learner`` role after a user has been linked. The learner portal has
+  logic to refresh the token if there are no roles included.
+* The generated link would point to the learner portal search by default but having the ``enterprise_customer_key`` should allow a user to be linked
+  from any page. We might want to redirect a user back to the search afterwards and strip the key from the url.
+* Although key usage can be limited, users might still get unintended access to an ``enterprise_customer_key`` and link themselves to an enterprise. 
+  The admin portal will add a feature to list all of the learners associated with an enterprise and allow admins to manually remove them.
+
+Phase 2 - Browsing anonymously
+==============================
+
+To support anonymous browsing, we will create a public page on the learner portal. Anyone with a universal link will be able to browse 
+an enterprise catalog and request course access on this page. The learner portal will make calls to new/modified endpoints to fetch data and display the catalog.
+The phase 1 implementation enables users to be associated with an enterprise which is required for requesting course access.
+Note that the anonymous browsing component could be built before the phase 1 implementation if it doesn't the include course access request.
+
+The process of generating the universal link will remain largely the same as in phase 1. The link would point to this new page rather than the search page,
+ie. ``/catalog?enterprise_customer_key=key``. We will also validate the ``enterprise_customer_key`` in the query params before rendering the page.
+Revoking a link will also be the same process as in phase 1.
+
+Since the learner portal makes calls to protected endpoints, we have to add/modify them to support the new public catalog page.
+The following are the APIs that the learner portal interacts with to display the enterprise catalog/course information:
+
+- edx-enterprise ``GET /enterprise-customer/``
+  * Fetches enterprise customer data such as uuid, enterprise_customer_catalogs, etc. by the enterprise slug.
+  * This is a protected endpoint.
+  * There is a lite version ``GET /enterprise-customer/basic_list`` that currently returns only id and name of an enterprise. 
+    We could modify this to also include the enterprise_customer_catalogs and that will be the minimal information we need to render the page.
```

Comment:
Lets firm up the language here and make a decision. I'm a +1 to modifying the `GET /enterprise-customer/basic_list` endpoint.